packages/babel-standalone/Gulpfile.js

```diff
@@ -100,24 +103,20 @@ function webpackBuild(filename, libraryName, version) {
   });*/
 }
 
-function registerGulpTasks(gulp) {
-  gulp.task("build-babel-standalone", ["build"], cb => {
-    pump(
-      [
-        gulp.src(__dirname + "/index.js"),
-        webpackBuild(
-          "babel.js",
-          "Babel",
-          require(__dirname + "/../package.json").version
-        ),
-        gulp.dest(__dirname + "/../"),
-        uglify(),
-        rename({ extname: ".min.js" }),
-        gulp.dest(__dirname + "/../"),
-      ],
-      cb
-    );
-  });
-}
-
-module.exports = registerGulpTasks;
+gulp.task("build", cb => {
```

Comment:
I'd prefer to still export this as a reusable function, as `babili-standalone` / `babel-minify-standalone` will reuse it. Maybe keep it in a separate file, and have a `Gulpfile.js` in this directory that imports it?